infor-design/enterprise
# 758 - Fix duplicate values when using paging

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Fixes duplicated values when paging is activated in `Lookup`. Changing the `value` from string to array so it can easily filter the duplicate values.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise-ng/issues/758

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Navigate to http://localhost:4000/components/lookup/test-paging-with-count.html
- Open the lookup
- Go to page 2
- Select any `Product ID`
- Then click `Apply` button
- Open again the lookup, then click `Apply`
- It should not duplicate the values in the input field
- Test also on multiple values
- Open lookup, go to any pages except page 1, then select more than 1 item
- Click `Apply` button
- It should show the original values, and should not duplicate any of those.
- Count should correct also
 
**Included in this Pull Request**:
